README.md

```diff
@@ -34,20 +34,25 @@ npm uninstall --save-dev ember-suave
 
 The `ember-cli-eslint` addon blueprint generates a `.eslintrc.js` configuration file at the root of the project. By default, it is set to extend ESLint's recommended subset of [core linting rules](http://eslint.org/docs/rules/).
 
-Add the plugin's
-[`recommended`](https://github.com/DockYard/eslint-plugin-ember-suave/blob/master/config/recommended.js) configuration to the list of extensions:
+Add the _plugin_ and its
+[`recommended`](https://github.com/DockYard/eslint-plugin-ember-suave/blob/master/config/recommended.js) configuration to the list of _extensions_:
 
 ```js
 // .eslintrc.js
 
 module.exports = {
   // ...
+  plugins: [
+    'ember',
+    'ember-suave'
+  ],
```

Comment:
28 days ago I got bit by this when adding ember-suave to an existing addon I had.  Then a couple days ago I bumped into this problem again, but was able to quickly fix it.  This might make the documentation a little more complete.